docs/user_guide/build_intel_boot_image.rst

```diff
@@ -0,0 +1,852 @@
+   $cp socfpga_arria10_socdk_ad9081/socfpga_arria10_socdk_sdmmc.dtb .
+   $cp socfpga_arria10_common/zImage .
+
+Writing the boot preloader partition requires special attention,
+first look for the device with BOOT mountpoint and annotate the third partition
+from the same device:
+
+.. code-block:: bash
+
+   lsblk
+
+Then, clear the partition with zeros and write the preloader image
+(in this example, Arria10 SoC's *./u-boot-splx4.sfp*):
+
+.. shell::
+   :no-path:
+
+   $DEV=mmcblk0p3
+   $cd /mnt/BOOT/socfpga_arria10_socdk_ad9081
+   $sudo dd if=/dev/zero of=/dev/$DEV oflag=sync status=progress \
+   $    bs=$(sudo blockdev --getsize64 /dev/$DEV) count=1
+    1+0 records in
+    1+0 records out
+    8388608 bytes (8.4 MB, 8.0 MiB) copied, 0.359183 s, 23.4 MB/s
+   $sudo dd if=./u-boot-splx4.sfp of=/dev/$DEV oflag=sync status=progress bs=64k
+    1697+1 records in
+    1697+1 records out
+    868996 bytes (869 kB, 849 KiB) copied, 0.21262 s, 4.1 MB/s
+
+.. tip::
+
+   The snippet below can infer the device based on the *BOOT* partition
+   mountpoint
+
+   .. shell::
+
+      $DEV=$(lsblk | sed -n 's/.*\(\b[s][d-z][a-z][0-9]\)\s*.*\/BOOT/\1/p' | sed 's/^\(...\).*/\1/')
+      $if [ -z "$DEV" ] ; then \
+      $   echo BOOT not found, couldn\'t infer block device ; \
+      $else \
+      $   echo The preloader image partition path likely is /dev/"$DEV"3 ; \
+      $fi
+
+Examples of building the boot image
+-------------------------------------------------------------------------------
+
+This is a list of projects supported by us for each carrier. The purpose is to
+illustrate how to build the different files involved in the process. Each
+project has its own characteristics (some files may differ from one project to
+the other).
+
+.. note::
+
+   Each project has its own Linux Kernel Image & Devicetree which needs to be
+   built. Follow these instructions to write the file to your SD card, depending
+   on the operating system that you use (Windows or Linux):
+
+   -  :dokuwiki:`[Wiki] Building the Intel SoC-FPGA kernel and devicetrees from source <resources/tools-software/linux-build/generic/socfpga>`
+   -  :dokuwiki:`[Wiki] Linux Download and setting up the image <resources/tools-software/linux-software/zynq_images/linux_hosts>`
+   -  :dokuwiki:`[Wiki] Formatting and Flashing SD Cards using Windows <resources/tools-software/linux-software/zynq_images/windows_hosts>`
+
+Proceed by cloning the repository, setting the environment to an ARM architecture
+cross compiler, build the configuration file, build the Kernel image, and
+lastly build the device tree (specific to each combination of carrier and eval
+board).
+
+You may notice that in the ``export CROSS_COMPILE`` examples there is a
+"trailing" dash ``-``, and the reason for this is because an export:
+
+.. shell::
+
+   $export CROSS_COMPILE=/path/to/arm-linux-gnueabihf-
+
+Within the Makefiles, this path becomes /path/to/arm-linux-gnueabihf-gcc
+(with ``gcc`` added at the end).
```

Comment:
```suggestion
"trailing" dash ``-``. That is because within the Makefiles, this path becomes
/path/to/arm-linux-gnueabihf-gcc (with ``gcc`` appended).

.. shell::

   $export CROSS_COMPILE=/path/to/arm-linux-gnueabihf-

```